modules/understanding-quick-start-elements.adoc

```diff
@@ -147,7 +147,7 @@ image::quick-start-icon.png[quick start icon element in the web console]
 
     - Kubernetes health probes and integration with Spring Actuator
 
-    - Metrics: Prometheus, Grafana, and integration with Spring Cloud Sleuth
+    - Metrics: Prometheus and integration with Spring Cloud Sleuth
```

Comment:
no need to remove, since it's related to openshift-monitoring grafana

Reply:
Agreed the Grafana mentioned here isn't related to the Grafana shipped by the cluster monitoring operator :)

Reply:
fixed